pycbc/waveform/waveform.py

```diff
@@ -852,6 +852,22 @@ def imrphenomhm_length_in_time(**kwargs):
     kwargs['f_lower'] = flow * 2./maxm
     return get_imr_length("IMRPhenomD", **kwargs)
 
+def seobnrv4hm_length_in_time(**kwargs):
```

Comment:
Can this be combined with the imr phenom one to avoid code duplication?

Reply:
Sure, let me see what I can do.

Reply:
@ahnitz just added a couple of changes. I've defined a function `get_hm_length_in_time` that takes input the (2,2) approximant name for the relevant HM approximant (`lor_approximant`) and its default `max ` and the makes the call to `lalsimulation.get_imr_length`.